src/styles/_catalog.scss

```diff
@@ -15,8 +32,4 @@
     border-radius: 5px;
     margin-bottom: 48px;
```

Comment:
With this margin bottom, empty space between each row equals 96px (gap + margin), gap only should be enough.